odoo_sh/advanced/containers.rst

```diff
@@ -186,3 +186,37 @@ Look for "*odoo: addons paths*":
 **Be careful**, especially with your production database.
 Operations that you perform running this Odoo server instance are not isolated:
 Changes will be effective in the database. Always, make your tests in your staging databases.
+
+Debugging in Odoo SH
+====================
+
+You can use pdb or pudb to debug your code on Odoo SH.
+As the server is run outside a shell, you cannot do it directly from your Odoo instance backend as the debugger needs a shell to operate.
+
+pdb is installed by default on all containers. If you want to use `pudb <https://pypi.org/project/pudb/>`_, you have to install it before:
+
+.. code-block:: bash
+
+  $  pip install pudb --user
```

Comment:
or simpler: add pudb to the requirements.txt (even if it does not look professional in a production project ^^)